_config.yml

```diff
@@ -1,6 +1,9 @@
 # This file is *NOT* reloaded automatically when you use 'jekyll serve'.
 # If you change this file, please restart the server process.
 
+# Options
+exclude: [vendor]
```

Comment:
That folder never gets generated in development, though it may make sense for clarity's sake.